src/analyz/mod.rs

```diff
@@ -658,14 +725,50 @@ fn emit_static_decl<'tcx>(
         "name": name,
         "ty": ty.to_json(ms),
         "mutable": mutable,
+        "kind": "body",
     });
     out.emit(EntryKind::Static, j)?;
-    emit_new_types(ms, out)
+    emit_new_defs(ms, out)
 }
 
 
 fn has_test_attr(tcx: TyCtxt, def_id: DefId) -> bool {
-    def_id.is_local() && tcx.has_attr(def_id, Symbol::intern("crux_test"))
+    if !def_id.is_local() {
+        return false;
+    }
+
+    for attr in tcx.get_attrs_unchecked(def_id) {
+        let crux = Symbol::intern("crux");
+        let test = Symbol::intern("test");
+        match &attr.kind {
```

Comment:
```suggestion
    let crux = Symbol::intern("crux");
    let test = Symbol::intern("test");
    for attr in tcx.get_attrs_unchecked(def_id) {
        match &attr.kind {
```
No sense re-interning the same constant strings with every iteration